package.json

```diff
@@ -61,7 +62,9 @@
     "kosmos-schemas": "^2.0.0",
     "kredits-contracts": "^5.4.0",
     "loader.js": "^4.7.0",
+    "ndjson": "^1.5.0",
```

Comment:
They use a frok: https://github.com/ipfs/js-ipfs-http-client/blob/master/package.json#L67

Reply:
Any idea why I have to add the package at all when it's a dependency of a dependency?
I didn't have any error about it locally, but Travis did.

Reply:
No. But the `kredits-contracts` dependencies are a mess. All kind of forks and beta releases... The blockchain community has to learn a lot about npm dependencies...

Reply:
Aragon npm deps:

![fail-imminent](https://user-images.githubusercontent.com/842/61233221-56521700-a730-11e9-9233-fcba4703ec5a.jpg)